packages/spanner/src/database.js

```diff
@@ -577,6 +593,25 @@ Database.prototype.run = function(query, options, callback) {
  *   .on('end', function() {});
  *
  * //-
+ * // If you need to enforce a specific param type, a types map can be provided.
+ * // This is typically useful if your param value can be null.
```

Comment:
No, when I wrote the tests for querying the various param types, I essentially copied the tests from the Java repo, in doing so certain queries will assume a type (in my case int64) and would throw an error if the param was not that same type. For whatever reason specifying the type made everything happy.